.github/workflows/ci-build.yml

```
@@ -0,0 +1,22 @@
+name: CI Build
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)